### ISSUE TITLE
Systems with sudoers config "always_set_home" can't find test_config.json file

### ISSUE THREAD

**Issue by bgurney-rh** [stratis-storage/ci#24]
The "test_config.json" file is created by a user running the stratis CI tests.  It is created by the administrator of the system to run the test, and contains a JSON list of the block devices that are clear for use in certain tests (currently the "stratisd" test, and in the future, the "blackbox" tests.)

A quick example of the contents of the test_config.json file is here, for a system with the operating system installed on /dev/vda, and spare test devices /dev/vdb, /dev/vdc, /dev/vdd, /dev/vde:

```
{
    "ok_to_destroy_dev_array_key": [
    				   "/dev/vdb",
    				   "/dev/vdc",
    				   "/dev/vdd",
    				   "/dev/vde"
    ]
}
```

The current standard location of the test_config.json is "in the home directory of the user calling the 'runalltests.sh' script."  However, if the script is being called via "sudo -E" to preserve environment variables, the $HOME environment variable will not be preserved if the system's /etc/sudoers file is configured with "Defaults always_set_home".

At this point, it seems best to relocate the standard position of the test_config.json file to a path that won't be affected by the inability to preserve environment variables.  One potential path would be "/etc/stratis/test_config.json".

(NOTE: this issue will also help to track when the new location of the test_config.json file is chosen, and enabled in the test scripts.)

**Comment by mulkieran** [comment]
The proposed change seems right to me.

**Comment by bgurney-rh** [comment]
This should be resolved, now that #25 is merged.

From this point forward, systems running the CI tests should save the list of test devices to "/etc/stratis/test_config.json".
